stdlib/sys/__init__.pyi

```diff
@@ -56,23 +56,24 @@ ps2: object
 
 # TextIO is used instead of more specific types for the standard streams,
 # since they are often monkeypatched at runtime. At startup, the objects
-# are initialized to instances of TextIOWrapper.
+# are initialized to instances of TextIOWrapper, but can also be None under
+# some circumstances.
 #
 # To use methods from TextIOWrapper, use an isinstance check to ensure that
 # the streams have not been overridden:
 #
 # if isinstance(sys.stdout, io.TextIOWrapper):
 #    sys.stdout.reconfigure(...)
-stdin: TextIO
-stdout: TextIO
-stderr: TextIO
+stdin: TextIO | MaybeNone
+stdout: TextIO | MaybeNone
+stderr: TextIO | MaybeNone
```

Comment:
Is there are reason why these are `MaybeNone` instead of `None`? I expect running mypy on `sys.stdout.write("...")` without an appropriate `if sys.stdout is not None` guard to tell me of the `NoneType has no attribute write` error that I would get from running that snippet of code using `pythonw.exe`. But instead it tells me that everything is fine.